tests/test.spec.js

```diff
@@ -0,0 +1,74 @@
+import { createStore } from 'redux';
+import { expect } from 'chai';
+import addReducer from '../index';
+import profileReducer from './reducers/profile.reducer';
+import cartReducer from './reducers/cart.reducer';
+
+function testAddNameReducer(store) {
+    store.dispatch({
+        type: 'ADD_NAME',
+        name: 'Rick Sanchez'
+    });
+
+    expect(store.getState().profile).to.deep.equal({
+        name: 'Rick Sanchez'
+    });
+}
+
+describe('Redux Store', () => {
+    let store;
+    beforeEach(() => {
+        store = createStore((state) => state);
+    });
+
+    it('should be an empty store', () => {
+        expect(store.getState()).to.equal(undefined);
```

Comment:
is it by design of `redux` or do you force that ?

Reply:
By design from `redux` this test was to be sure the store was empty from the start but it's not a usefull one. Removing it